docs/snippets/all/tutorials/send_columns_arrays.cpp

```diff
@@ -0,0 +1,53 @@
+// Use the `send_columns` API to send several point clouds over time in a single call.
+
+#include <array>
+#include <rerun.hpp>
+#include <vector>
+
+using namespace std::chrono_literals;
+
+int main() {
+    const auto rec = rerun::RecordingStream("rerun_example_send_columns_arrays");
+    rec.spawn().exit_on_failure();
+
+    // 5 point clouds each with 4 points.
+    std::vector<std::array<float, 3>> positions = {
+        // clang-format off
+        {1.0, 0.0, 1.0}, {0.5, 0.5, 2.0},
+        {1.5, -0.5, 1.5}, {1.0, 1.0, 2.5}, {-0.5, 1.5, 1.0}, {-1.5, 0.0, 2.0},
+        {2.0, 0.0, 2.0}, {1.5, -1.5, 3.0}, {0.0, -2.0, 2.5}, {1.0, -1.0, 3.5},
+        {-2.0, 0.0, 2.0}, {-1.5, 1.5, 3.0}, {-1.0, 1.0, 3.5},
+        {1.0, -1.0, 1.0}, {2.0, -2.0, 2.0}, {3.0, -1.0, 3.0}, {2.0, 0.0, 4.0},
```

Comment:
Is each row one cloud? If so, they have varying number of points, not 4 each?

Reply:
Actually, isn't this the same point cloud that's changing over time?

Reply:
hmm yeah that commit is a tad bit misleading!

Reply:
all of these comments are pretty bad Oo. Got late when I did this :|